test/suites/integration/expiration_test.go

```diff
@@ -71,7 +75,7 @@ var _ = Describe("Expiration", func() {
 			g.Expect(ok).To(BeTrue())
 		}).Should(Succeed())
 
-		env.EventuallyExpectCreatedNodeCount("==", 2)
+		env.EventuallyExpectCreatedNodeCount("==", 1)
```

Comment:
Why should this be one? Shouldn't it still be two since we're creating the replacement node?

Reply:
No, we reset the counter, so expect to just create one nodeclaim

Reply:
Oh, missed the monitor reset. Looks good then 👍